coralproject/talk
# fix: handle 'truthy value with falsey url' edgecase in media input

## What does this PR do?
This PR addresses https://vmproduct.atlassian.net/browse/CORL-2176 by handling the edge case in which the external media input has a non null Value, but that value has an empty `url` field. This occurs when the user clicks "Insert" before typing anything into the input.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
1. Enable external media
2. Click the "+" button on the comment form
3. click the "Insert" button
4. Observe that the input does not close
